Jenkinsfile

```diff
@@ -1,13 +1,13 @@
 // This file serves as working example and tests pipeline library functionality at same time.
```

Comment:
I would also remove these comments and then remove `repository-validator.json` because there is no longer a need to ignore `HasNewestPtcsJenkinsLibRule`